Client/ApiClient.php

```diff
@@ -127,24 +128,33 @@ protected function getResponseData(
         Request $request
     ): ?string {
         $this->logger->debug(
-            "API Request:{$request->getMethod()} {$request->getUri()}",
```

Comment:
But why? I disagree, the context might be lost depending of the log formatter

Reply:
Ok we can keep this in the log itself if you prefer

Reply:
Yup